developer-docs-site/docs/guides/move-guides/move-on-aptos.md

```diff
@@ -38,7 +38,7 @@ The Aptos Move adapter features include:
 The Aptos framework ships with many useful libraries:
 * A [Token standard](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/token.move) that makes it possible to create NFTs and other rich tokens without publishing a smart contract
 * A [Coin standard](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/coin.move) that makes it possible to create type-safe Coins by publishing a trivial module
-* An [iterable Table](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/sources/iterable_table.move) that allows for traversing all the entries within a table
+* An [iterable Table](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/data_structures/sources/iterable_table.move) that allows for traversing all the entries within a table
```

Comment:
I would remove `iterable-table` from this list as it's not part of the framework

Reply:
Done.